test/src/main/java/io/strimzi/test/extensions/StrimziExtension.java

```diff
@@ -93,9 +93,18 @@
     private static final String START_TIME = "start time";
     private static final String TEST_LOG_DIR = System.getenv().getOrDefault("TEST_LOG_DIR", "../systemtest/target/logs/");
 
-    /** Tags */
+    /**
+     * Tag for acceptance tests, which are triggered for each push/pr/merge on travis-ci
+     */
     public static final String ACCEPTANCE = "acceptance";
+    /**
+     * Tag for regression tests which are stable.
+     */
     public static final String REGRESSION = "regression";
+    /**
+     * Tag for tests, which's results are not 100% reliable on all testing environments.
+     */
+    public static final String FLAKY = "flaky";
```

Comment:
So would be be useful to split this up according to the test environment, allowing us to only exclude flaky tests on the environment where they were flaky. That would increase coverage on platforms where those tests were not flaky, which must be a good thing, right?